pkg/annotations/dynamic-network-status.go

```diff
@@ -0,0 +1,79 @@
+package annotations
+
+import (
+	"encoding/json"
+	"fmt"
+
+	corev1 "k8s.io/api/core/v1"
+
+	nettypes "github.com/k8snetworkplumbingwg/network-attachment-definition-client/pkg/apis/k8s.cni.cncf.io/v1"
+	multusapi "gopkg.in/k8snetworkplumbingwg/multus-cni.v3/pkg/server/api"
+)
+
+func AddDynamicIfaceToStatus(currentPod *corev1.Pod, networkSelectionElement *nettypes.NetworkSelectionElement, response *multusapi.Response) (string, error) {
+	currentIfaceStatus, err := podDynamicNetworkStatus(currentPod)
+	if err != nil {
+		return "", err
+	}
+
+	newIfaceStatus := nettypes.NetworkStatus{
+		Name: fmt.Sprintf("%s/%s", networkSelectionElement.Namespace, networkSelectionElement.Name),
+	}
+
+	if response != nil && response.Result != nil {
+		for _, iface := range response.Result.Interfaces {
+			newIfaceStatus.Interface = iface.Name
+			newIfaceStatus.Mac = iface.Mac
+		}
+
+		var ips []string
+		for _, ifaceIPConfig := range response.Result.IPs {
+			ips = append(ips, ifaceIPConfig.Address.String())
+		}
+		if len(ips) > 0 {
+			newIfaceStatus.IPs = ips
+		}
+	}
+
+	newIfaceString, err := json.Marshal(append(currentIfaceStatus, newIfaceStatus))
+	if err != nil {
+		return "", fmt.Errorf("failed to marshall the dynamic networks status")
+	}
+	return string(newIfaceString), nil
+}
+
+func DeleteDynamicIfaceFromStatus(currentPod *corev1.Pod, netName string, ifaceName string) (string, error) {
```

Comment:
Why we don't pass the netName as `NetworkSelectionElement` like at the Add counterpart then we use 

We have the `namespacedName(networkSelectionElement.Namespace, networkSelectionElement.Name)`  function

Reply:
I mean changet that function to have the same time for netName as "AddDyn..."